src/server/game/Loot/LootMgr.cpp

```diff
@@ -404,7 +404,7 @@ LootItem::LootItem(LootStoreItem const& li)
 }
 
 // Basic checks for player/item compatibility - if false no chance to see the item in the loot
-bool LootItem::AllowedForPlayer(Player const* player, bool isGivenByMasterLooter /*= false*/, bool allowQuestLoot /*= true*/, ObjectGuid source) const
+bool LootItem::AllowedForPlayer(Player const *player, ObjectGuid source) const
```

Comment:
```suggestion
bool LootItem::AllowedForPlayer(Player const* player, ObjectGuid source) const
```
always this way

Reply:
For more code standards you can read here https://www.azerothcore.org/wiki/cpp-code-standards